test/user-agent/test/common.js

```diff
@@ -64,12 +66,17 @@ export const trustBlocStartupOpts = {
     walletMediatorURL: 'http://localhost:10093'
 }
 
+export async function loadAries(name) {
+    let opts = ariesStartupOpts
+    if(name) {
+        opts = JSON.parse(JSON.stringify(ariesStartupOpts))
+        opts["db-namespace"] = `${name}db`
+        opts["agent-default-label"] = `${name}-user-agent`
```

Comment:
Object attributes that have to be enclosed in double quotes are not javascript-friendly. We should stop adding more of those.